xbmc/video/GUIViewStateVideo.cpp

```diff
@@ -181,6 +181,7 @@ CGUIViewStateWindowVideoNav::CGUIViewStateWindowVideoNav(const CFileItemList& it
       }
       break;
     case NODE_TYPE_TAGS:
+    case NODE_TYPE_VIDEOVERSIONS:
```

Comment:
Here you seem to have added a method of navigating to the video versions but it's incomplete.  There doesn't seem to be a way to reach this point in the same way that I can for sets, tags, directors actors etc.  The only way to see the different versions seems to be the new dialog.

So either the node is not needed or the navigation needs finishing off.